modules/unsupported/mbtiles/src/main/java/org/geotools/mbtiles/MBTilesMetadata.java

```diff
@@ -155,7 +191,11 @@ public void setBoundsStr(String boundsStr) {
             double miny = Double.parseDouble(matcherEnvelope.group(2));
             double maxx = Double.parseDouble(matcherEnvelope.group(3));
             double maxy = Double.parseDouble(matcherEnvelope.group(4));
-            setBounds(new ReferencedEnvelope(minx, maxx, miny, maxy, null));
```

Comment:
The exception happens if "EPSG:4326" can't be parsed to a CRS.  I think that should never actually happen.